aiohttp_client_cache/cache_control.py

```diff
@@ -215,13 +215,24 @@ def get_refresh_headers(
     return conditional_request_supported, refresh_headers
 
 
-def parse_http_date(value: str) -> datetime | None:
-    """Attempt to parse an HTTP (RFC 5322-compatible) timestamp"""
-    try:
-        return parsedate_to_datetime(value)
-    except (TypeError, ValueError):
-        logger.debug(f'Failed to parse timestamp: {value}')
-        return None
+if sys.version_info >= (3, 10):
```

Comment:
Could you explain this change? Did behavior for `parsedate_to_datetime()` change in 3.10? I couldn't find anything about it in the [3.10 changelog](https://docs.python.org/3/whatsnew/3.10.html).

Reply:
Yes, the function has been changed since Python 3.10.  

Here is a test where `TypeError` is raised only on Python 3.8 and 3.9: https://github.com/requests-cache/aiohttp-client-cache/actions/runs/7542785272/job/20532378980

Looking at the source code we can see the difference:
- Python 3.8 https://github.com/python/cpython/blob/bf477a99e0c85258e6573f4ee9eda68fa1f98a31/Lib/email/utils.py#L197
- Python 3.10 https://github.com/python/cpython/blob/4a32275389d94ba41f8881c32ad4b232effb1c6f/Lib/email/utils.py#L315


Since Python 3.10 it is explicitly documented that `ValueError` is raised: https://docs.python.org/3.10/library/email.utils.html?highlight=parsedate_to_datetime#email.utils.parsedate_to_datetime

Reply:
Good catch. Thanks for the explanation.